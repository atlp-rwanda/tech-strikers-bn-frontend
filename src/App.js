import React from "react";

const App = () => (
  <div>Hello! This Barefoot Nomad created from Scratch. WOW</div>
);

export default App;
