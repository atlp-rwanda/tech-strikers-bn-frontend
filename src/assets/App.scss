* {
  margin: 0px;
  padding: 0px;
}
.modal {
  width: 105%;
}
.modal-content {
  width: 80%;
}
.login-btn,
.signup-btn {
  background-color: rgba(255, 255, 255, 0);
  color: #fff;
  border: 0px;
}
.login-btn::after,
.signup-btn::after {
  border: none;
}
.login-submit {
  width: 100%;
  height: 40px;
  padding: 0px;
  margin-top: 15px;
}
.reset-pass-link {
  text-decoration: none;
}
.icon-login-div {
  margin-left: 45%;
}
.authIcon {
  display: block;
  width: 20%;
}
.close {
  width: 20%;
  border: none;
  margin-left: 85%;
  -webkit-appearance: button;
}
.starIcon {
  width: 8%;
}
.locationIcon {
  width: 10%;
  height: 10%;
  margin: 0px;
  padding: 0px;
}
.cardBox {
  width: 85%;
  margin-right: 30%;
}
.cardContent {
  padding: 7%;
}
.row.content {
  height: 450px;
}

.sidenav {
  padding-top: 20px;
  background-color: #f1f1f1;
  height: 100%;
}

/* Set black background color, white text and some padding */
footer {
  background-color: #555;
  color: white;
  padding: 15px;
}

/* On small screens, set height to 'auto' for sidenav and grid */
@media screen and (max-width: 767px) {
  .sidenav {
    height: auto;
    padding: 15px;
  }
  .row.content {
    height: auto;
  }
}
.error {
  border: 2px solid #ff6565;
}
