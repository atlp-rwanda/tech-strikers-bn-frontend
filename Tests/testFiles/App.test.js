import React from "react";
import App from "../../src/App";
import renderer from "react-test-renderer";
import { cleanup, render } from "@testing-library/react";

afterEach(cleanup);

it("snapshot test", () => {
  const appTest = renderer.create(<App />).toJSON();
  expect(appTest).toMatchSnapshot();
});

// it("render component", () => {
//   const component = render(<App />);
//   console.log(component);
// })
