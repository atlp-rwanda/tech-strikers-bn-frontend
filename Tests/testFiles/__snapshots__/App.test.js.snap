// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div>
  <div
    className="d-flex bd-highlight p-2 text-white"
    style={
      Object {
        "background": "#061437",
      }
    }
  >
    <div
      className="mr-auto flex-grow-1 p-2 bd-highlight"
    >
      Barefoot Nomad
    </div>
    <button
      className="p-2 bd-highlight signup-btn"
    >
      Signup
    </button>
    <button
      className="ml-auto p-2 bd-highlight login-btn"
      onClick={[Function]}
    >
      Login
    </button>
  </div>
  <section>
    <img
      className="topSection"
      src="test-file-stub"
      style={
        Object {
          "maxHeight": "500px",
        }
      }
    />
  </section>
  <div
    className="d-flex flex-row bd-highlight justify-content-center mb-3"
    style={
      Object {
        "backgroundColor": "#061437",
      }
    }
  >
    <div
      className="text-white  mx-auto bd-highlight"
    >
      <p
        className="p-3"
      >
        Where do you want to go?
      </p>
    </div>
    <div
      className="p-2  mx-auto bd-highlight"
    >
      <button
        className="text-white btn btn-outline-dark"
        disabled={false}
        type="button"
      >
        Destination
      </button>
       
    </div>
    <div
      className="p-2  mx-auto bd-highlight"
    >
      <button
        className="w-100 d-inline-block text-white btn btn-outline-dark"
        disabled={false}
        type="button"
      >
        Date
      </button>
       
    </div>
    <div
      className="p-2  mx-auto bd-highlight"
    >
      <button
        className="text-white btn btn-outline-dark"
        disabled={false}
        type="button"
      >
        Travel Type
      </button>
       
    </div>
    <div
      className="p-2  mx-auto bd-highlight"
    >
      <button
        className="text-white btn btn-danger"
        disabled={false}
        type="button"
      >
        Search
      </button>
       
    </div>
  </div>
  <section
    style={
      Object {
        "paddingBottom": "20px",
        "paddingTop": "20px",
      }
    }
  >
    <div
      className="container"
    >
      <h2
        className="text-center"
      >
        Most Visited Places
      </h2>
      <div
        className="row"
      >
        <div
          className="col-4"
        >
          <div
            className="card shadow-lg mb-5 bg-blue rounded cardBox"
          >
            <img
              alt="hero"
              className="card-img-top "
              src="test-file-stub"
            />
            <div
              className="cardContent"
            >
              Dar-es-Salam,
              <div>
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
              </div>
              <div
                style={
                  Object {
                    "paddingTop": "15px",
                  }
                }
              >
                <img
                  className="starIcon locationIcon"
                  src="test-file-stub"
                />
                Tanzania
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-4"
        >
          <div
            className="card shadow-lg mb-5 bg-blue rounded cardBox"
          >
            <img
              alt="hero"
              className="card-img-top "
              src="test-file-stub"
            />
            <div
              className="cardContent"
            >
              Kigali,
              <div>
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
              </div>
              <div
                style={
                  Object {
                    "paddingTop": "15px",
                  }
                }
              >
                <img
                  className="starIcon locationIcon"
                  src="test-file-stub"
                />
                Rwanda
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-4"
        >
          <div
            className="card shadow-lg mb-5 bg-blue rounded cardBox"
          >
            <img
              alt="hero"
              className="card-img-top "
              src="test-file-stub"
            />
            <div
              className="cardContent"
            >
              Nairobi,
              <div>
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
                <img
                  className="starIcon"
                  src="test-file-stub"
                />
              </div>
              <div
                style={
                  Object {
                    "paddingTop": "15px",
                  }
                }
              >
                <img
                  className="starIcon locationIcon"
                  src="test-file-stub"
                />
                Kenya
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <footer
    style={
      Object {
        "background": "#061437",
        "color": "white",
        "paddingBottom": "25px",
        "paddingLeft": "20px",
        "paddingTop": "50px",
      }
    }
  >
    <section
      className="container"
    >
      <div
        className="row"
      >
        <main
          className="col-4"
        >
          <h3>
            Barefoot Nomad
          </h3>
          <p
            style={
              Object {
                "color": "rgba(255,255,255,0.81)",
              }
            }
          >
            Simplifying employee travels
          </p>
          <div>
            <p
              style={
                Object {
                  "color": "#5E6D77",
                  "marginBottom": "0px",
                }
              }
            >
              KN 54 St,
               
            </p>
            <p
              style={
                Object {
                  "color": "#5E6D77",
                }
              }
            >
              Kigali, Rwanda
            </p>
          </div>
        </main>
        <main
          className="col-4 mt-4 text-center contactMain"
        >
          <p>
            Accomodation
          </p>
          <p>
            Most Visited place
          </p>
        </main>
        <main
          className="col-4 text-center contactMain"
        >
          <h4>
            Contact us
          </h4>
          <p
            className="text-center"
          >
            Email: strikers668@gmail.com
          </p>
        </main>
      </div>
      <section
        className="text-center"
        style={
          Object {
            "color": "#BCBCBC",
          }
        }
      >
        Â© 
        2021
         Barefoot Nomad By
         
        <span
          style={
            Object {
              "color": "#0085FF",
            }
          }
        >
          TeckStrickers
        </span>
      </section>
    </section>
  </footer>
</div>
`;
